_posts/2013-04-02-editors.md

```diff
@@ -27,3 +27,35 @@ If you are using [SublimeText 3](http://www.sublimetext.com) with `ember-cli`, b
 // project basis.
 "folder_exclude_patterns": [".svn", ".git", ".hg", "CVS", "tmp/class-*", "tmp/es_*", "tmp/jshinter*", "tmp/replace_*", "tmp/static_compiler*", "tmp/template_compiler*", "tmp/tree_merger*", "tmp/coffee_script*", "tmp/concat-tmp*", "tmp/export_tree*", "tmp/sass_compiler*"]
 {% endhighlight %}
+
+### WebStorm
+If you are using [WebStorm](https://www.jetbrains.com/webstorm/) with `ember-cli`, you will need to modify your `.gitignore` file and `ECMAScript6` settings.
+
+First, add the following line to `.gitignore`:
+```
+.idea
+```
+
+Next, from the WebStorm menu:
+
+`File > Settings -> Languages & Frameworks -> JavaScript -> ECMAScript6`
+
+Right-click on each of the following directories and mark as indicated:
+
+Mark as `excluded`:
+```
```

Comment:
@twokul Thanks again, but which lines? I'm sure I'm missing some kind of marking.